src/test/add.e2e.test.ts

```diff
@@ -0,0 +1,69 @@
+import request from 'supertest';
+import jwt from 'jsonwebtoken';
+import app from '../app';
+import { connectDB, disconnectDB } from '../repository/mongoRepository';
+
+const generateTestToken = (userId: string, role: string) => {
```

Comment:
I would suggest creating a similar function to create the token when user logs in and them import and reuse it here